src/styles/fonts/_roboto.scss

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Regular.woff2) format('woff2');
+  src: url(../fonts/Roboto-Regular.ttf) format('truetype');
```

Comment:
The `src` property should include all font formats in a single declaration. Combine the `woff2` and `ttf` formats using the comma to separate them, to prevent the `ttf` format from being ignored as it is currently overridden by the second `src` declaration.